docs/root/install/tools/route_table_check_tool.rst

```diff
@@ -18,6 +18,10 @@ Input
      :ref:`config <config_tools_router_check_tool>`.
      The tool config input file specifies urls (composed of authorities and paths)
      and expected route parameter values. Additional parameters such as additional headers are optional.
+     Schema: The tool deals with JSON objects internally, based on this :repo:`schema<test/tools/router_check/json/tool_config_schemas.cc>`.
+     A recent feature moves all internal schemas in the tool to proto3. This feature can be enabled by an extra optional parameter ``--useproto``.
+     It introduces a breaking change in the test based on :repo:`schema<test/tools/router_check/validation.proto>`.
```

Comment:
Optimally we would figure out a way to actually run protodoc on this file so it shows up for real in the docs. This might be a little difficult based on where it is in the tree (not in the API tree) so OK to skip for now.

Reply:
@mattklein123 Yes I tried doing that, but I couldn't find examples for test protos in the tree.